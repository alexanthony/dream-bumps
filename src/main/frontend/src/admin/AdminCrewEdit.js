import React, {PropTypes} from 'react';
import {connect} from 'react-redux';
import {Card, CardTitle, CardText} from 'material-ui/Card';
import {Field, reduxForm} from 'redux-form';
import {TextField} from 'redux-form-material-ui';
import RaisedButton from 'material-ui/RaisedButton';
import {Cell, Grid} from 'material-grid/dist';
import Select from 'react-select';
import {SelectField} from 'redux-form-material-ui';
import MenuItem from 'material-ui/MenuItem';

import AdminCrewDetailsEdit from './AdminCrewDetailsEdit';
import AdminCrewMemberEdit from './AdminCrewMemberEdit';

const AdminCrewEdit = ({user, handleSubmit, params}) => (
  <Grid>
    <Cell col="6">
      <AdminCrewDetailsEdit crewId={params.crewId}/>
    </Cell>
    <Cell col="6">
      <AdminCrewMemberEdit crewId={params.crewId}/>
    </Cell>
  </Grid>
);

// todo - activated and authorities


export default AdminCrewEdit;
