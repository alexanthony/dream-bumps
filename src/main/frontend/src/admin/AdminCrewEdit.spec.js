// @flow
import React from 'react';
import { shallow } from 'enzyme';
import { AdminCrewEdit } from './AdminCrewEdit';
import toJson from 'enzyme-to-json';

test('Renders properly', () => {
  const component = shallow(
    <AdminCrewEdit
      params={{crewId: "17"}}
    />
  );
  expect(component).toBeDefined();
  expect(toJson(component)).toMatchSnapshot()
});
