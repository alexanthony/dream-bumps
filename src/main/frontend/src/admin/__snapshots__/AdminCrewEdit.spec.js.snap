exports[`test Renders properly 1`] = `
<Grid>
  <Cell
    col="6">
    <Connect(ReduxForm)
      crewId="17" />
  </Cell>
  <Cell
    col="6">
    <Connect(ReduxForm)
      crewId="17" />
  </Cell>
</Grid>
`;
